src/main/java/bgu/spl/net/impl/stomp/Client.java

```diff
@@ -9,6 +9,7 @@
     public String host;
     public String login;
     public String passcode;
+    public boolean connected;
```

Comment:
why public and not getters and setters?

Reply:
I did the same with everything in this class, should I change it?

Reply:
Yes